net.sourceforge.pdfchain.json

```diff
@@ -0,0 +1,84 @@
+{
+    "app-id": "net.sourceforge.pdfchain",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "40",
+    "sdk": "org.gnome.Sdk",
+    "command": "pdfchain",
+    "finish-args": [
+    	"--filesystem=xdg-documents",
+        "--filesystem=xdg-download",
+        "--filesystem=xdg-pictures",
+        "--filesystem=home:ro",
+        "--filesystem=host:ro",
+        "--share=ipc",
+        "--socket=x11",
+        "--socket=wayland",
+        "--device=dri"
+    ],
+    "cleanup": [
+        "/include",
+        "/lib/pkgconfig",
+        "/share/pkgconfig",
+        "/share/aclocal",
+        "/man",
+        "/share/man",
+        "/share/gtk-doc",
+        "/share/vala",
+        "*.la",
+        "*.a"
+    ],
+    "modules": [
+    	{
+            "name": "pdftk",
+            "buildsystem": "simple",
+            "build-commands": [
+            	"mkdir -p /app/bin",
+                "cp pdftk /app/bin/pdftk",
+                "chmod a+x /app/bin/pdftk"
+            ],
+            "sources": [
+                {
+                    "type": "file",
+                    "__comment1__": "This is a pre-built x86_64 native image of pdftk-java compiled with GraalVM, that's why",
+                    "__comment2__": "flathub build is x86_64 only, see https://gitlab.com/pdftk-java/pdftk#pre-built-binaries",
+                    "url": "https://gitlab.com/pdftk-java/pdftk/-/jobs/924565150/artifacts/raw/build/native-image/pdftk",
```

Comment:
why not build from source and use JDK? having aarch64 support would be nice.

Reply:
even graalvm has aarch64 binaries.

Reply:
That would be nice of course, but more work :-).. Future MR can add that source build support, or maybe a native image for aarch64 will also be released (same as x86_64) and we can use that.

Reply:
there is a .jar artifact next to that "native" image.